blip-00xx.md

```diff
@@ -0,0 +1,220 @@
+    * 2 byte `cltv_expiry_delta`
+    * 4 byte `fee_proportional_millionths`
+    * 4 byte `fee_base_msat`
+- `payment_constraints`:
+    * 4 byte `max_cltv_expiry`
+    * 8 byte `htlc_minimum_msat`
+- `allowed_features`: optional
+
+If we assume that the `allowed_features` vector is not set, then this comes to 
+a total of 30 mandatory bytes. 
+
+For the recipient node, it will receive a cipher text payload containing:
+
+- `padding`: optional
+- `path_id`: let's assume that this is 32 bytes like the existing payment 
+   address.
+- `payment_constraints`:
+    * 4 byte `max_cltv_expiry`
+    * 8 byte `htlc_minimum_msat`
+
+This comes to a total of 44 bytes for the recipient's cipher text. 
+
+The padding field should be used by recipients to pad cipher text blobs so that 
+all are the same size. Since the calculated recipient cipher text blob size (44) 
+is larger than that of the forwarding nodes (30), we can assume that all the
+cipher text blobs will have a size of around 44 bytes. 
+
+##### `blinded_hop` Size Estimation
+
+The total number of bytes required for a single `blinded_hop` is:
+
+    = 33+2+len(cipher_text)
+    = 35+len(cipher_text)
+
+If we use the estimated `cipher_text` size of 44 bytes, then this comes to 79 
+bytes for a single `blinded_hop`.
+
+##### `blinded_payinfo` Size Estimation
+
+The total number of bytes required for the encoding of a single 
+`blinded_payinfo` entry is:
+
+    = 4+4+2+8+8+2+len(features)+33+1+num_hops(len(blinded_hop))
+    = 68+len(features)+num_hops(len(blinded_hop))
+
+If we take the estimate of 79 bytes per `blinded_hop` and if we assume an empty 
+feature vector then this comes to:
+
+    = 68+num_hops(79)
+
+The maximum number of hops in a single blinded path can then be calculated to 
+be:
+
+    639 = 68+num_hops(79) 
+    num_hops = 7
+
+#### QR code limit
+
+Another soft maximum value to keep in mind is the maximum number of bytes that
+can fit into a [QR code][qr] which is 2,953 bytes. This is a soft maximum 
+because this only applies if the invoice is in fact being transmitted via QR 
+code. This limit does not apply if the invoice is being transmitted via other
+protocols such as LNURL. In the cases where the limit does apply, then two 
+variables will be at play: 
+
+- The number of blinded paths
+- The number of blinded hops within each path (which will always also be 
+  restricted by the `data_length` maximum).
+
+The exact limit on the number of blinded paths that can be included depends on
+the size of other fields in the invoice. It is worth noting that an invoice with
+a blinded path should not contain any `r` (route hint) fields.
+
+## Universality
+
+This proposal is a temporary measure that will allow users to start making use
+of blinded paths in the context of payments and thereby taking advantage of the 
+potential privacy and payment success rate benefits that in they will in theory 
+provide. Once the Offers protocol along with its new invoice format has been 
```

Comment:
link [Offers](https://github.com/lightning/bolts/pull/798)

Reply:
cool can add - tbh my assumption was always that if you've linked something once then you dont need to link it again so as not to litter the doc with blue links? but happy to update - just had this assumption